web/src/index.html

```diff
@@ -47,7 +47,7 @@ <h1 class="title">Game</h1>
           </div>
           <div class="game">
           <!-- Test game Canvas -->
-            <canvas id="gameCanvas" width="900" height="600" style="border:1px solid #000000; background-color:black;"></canvas>
+            <canvas id="gameCanvas" width="900" height="600"></canvas>
```

Comment:
I only have a question here, no opinion. Would we want to have a variable canvas size? Depending on the screen size.